brozzler/cli.py

```diff
@@ -234,6 +234,25 @@ def brozzle_page(argv=None):
         action="store_true",
         help="Try to avoid web bot detection",
     )
+    arg_parser.add_argument(
+        "--metrics_port",
+        type=int,
+        dest="metrics_port",
+        default=8888,
+        help="Port for brozzler's Prometheus scrape endpoint",
+    )
+    arg_parser.add_argument(
+        "--registry_url",
+        dest="registry_url",
+        default=None,
+        help="http-sd-registry url, for Prometheus metrics discovery",
+    )
+    arg_parser.add_argument(
+        "--env",
+        dest="env",
+        default=None,
+        help="env for Prometheus target registry",
```

Comment:
update help text here, maybe "deployment environment for this brozzler instance"